app/templates/admin/cceMinor.html

```diff
@@ -60,70 +60,146 @@ <h2>CCE Minor Progress</h2>
+                  role="tab" 
+                  aria-controls="{{ tab.id }}" 
+                  aria-selected="{% if tab.id == defaultActiveTab %}true{% else %}false{% endif %}">
+            {{ tab.label }}
+          </button>
+        </li>
+    {% endfor %}
+  </ul>
+  
+  <div class="tab-content mt-3" id="studentTabsContent">
+    <form id="updateMinorInterestForm" class="d-none" method="POST"></form>
+    
+    {% for tab in tabData %}
+        <div class="tab-pane fade {% if tab.id == defaultActiveTab %}show active{% endif %}" 
+             id="{{ tab.id }}" 
+             role="tabpanel" 
+             aria-labelledby="{{ tab.id }}-tab">
+          <table class="display" id="{{ tab.id }}StudentsTable">
+            <thead>
+              <tr>
+                <th class="w-50">Name</th>
+                <th class="text-center"></th>
+                <th class="text-center"></th>
+              </tr>
+            </thead>
+            <tbody>
+              {% for student in tab.students %}
+                <tr>
+                  <td class="w-50">
+                    <a href="/profile/{{student.username}}/cceMinor" target="_blank">
+                      {{student.firstName}} {{student.lastName}}
+                    </a>
+                  </td>
+                  <td class="text-center">
+                    <button class="updateMinorInterestButton btn btn-primary btn-sm" 
+                            data-url="/profile/{{student.username}}/updateMinorDeclaration" 
+                            id="{{student.username}}" 
+                            data-username="{{student.username}}">{{ tab.actionLabel }}
+                    </button>
+                  </td>
+                  <td class="text-center">
+                    <button type="button" 
+                            class="remove_minor_candidate btn btn-danger btn-sm" 
+                            id="{{student.username}}" 
+                            aria-label="remove minor candidate">Remove Student
+                    </button>
+                  </td>
+                </tr>
+              {% endfor %}
+              <input id="{{ tab.id }}StudentEmails" hidden value="{{ tab.emailString }}"/>
+            </tbody>
+          </table>
+        </div>
+    {% endfor %}
+  </div>
+</div>
+{% endmacro %}
 
 <!-- #################  Modal  ################# -->
 
-<div class="modal fade float-start" id="addInterestedStudentsModal" tabindex="-1" aria-labelledby="addInterestedStudentsLbl" aria-hidden="true">
+<div class="modal fade" id="addInterestedStudentsModal" tabindex="-1" aria-labelledby="addInterestedStudentsLbl" aria-hidden="true">
     <div class="modal-dialog">
       <div class="modal-content">
         <div class="modal-header">
-          <h5 class="modal-title">{{modalTitle}}</h5>
-          <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+          <h5 class="modal-title">Add Students Interested in a Minor</h5>
```

Comment:
I would like this modal to reflect which tab we are on if possible. Feels a little off to be viewing a list of declared students and when clicking on Add Students it is having you add Interested students instead. We can talk about it.